doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>to same predicted class -&gt; to the same predicted class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>